README.md

```diff
@@ -93,4 +93,67 @@ You could use the block-form usage of the form-field component with components t
 
 ## Using your own components
 
-You could use your own components by implementing it in the form-field component like above in the block-form usage. 
+You could use your own components by implementing it in the form-field component like above in the block-form usage.
+
+# Paper-form-field
+Paper-form-field is a variant of the default form-field, using ember-paper for Google Material Design like form-fields. The following sections descripe how the paper-form-field work and what options you can pass to it.
+
+## Usage
+```handlebars
+{{paper-form-field type="text-area"
+                   object=movie
+                   propertyPath="description"
+                   options=options
+                   updated=(action "update")
+                   disabled=locked}}
+```
+
+
+## Options
+Options is an object the can contain attributes and methods. Every attribute and method will be added to the component. This can be used to reach/supply build in abillities or add/overwrite event handlers.
+
+## htmlAttributes
+If you want to pass attributes to the actual html element that the form field renders, you can add 'htmlAttributes' to the options object. This object can contain thing like placeholder, name of maxlength. You can read the ember-paper documentation for more information.
+
+## Available types
+
+The following paper form field types are available.
+
+### number-field
+
+Renders a HTML-input[type="text"], wrapped by a component with attributes, event handlers and build-in methods. You can overwrite them by passing attributes/methods as options, as described above.
+
+**decimals** By how many decimals your number should be rounded. Default is 2.
+
+**keyDown(e)** By default you can use arrow-up and arrow-down to increase/decrease the number by one. It _won't_ round the number, so it's useable with floats.
+
+**formatOnChange(value)** Gets the value and formats its, before updating the object. By default it formats a number to '17 009,87'.
+
+#### Example
+
+```javascript
+options: {
+  decimals: 0
+}
+
+htmlAttributes: {
+  minValue: 0
+  maxValue: 99
+}
+
+```
+
+```handlebars
+{{paper-form-field type="number-field"
+                   object="bakery"
+                   propertyPath="baguettes"
+                   options=options
+                   htmlAttributes=htmlAttributes
```

Comment:
So, this example shows the `htmlAttributes` passed directly to the component, while the text above describes adding them to the `options` attribute. Which method is correct?